inc/class-statify-frontend.php

```diff
@@ -92,11 +92,23 @@ public static function track_visit( $is_snippet = false ) {
 		}
 
 		/* Relative target url */
-		$data['target'] = user_trailingslashit( str_replace( home_url( '/', 'relative' ), '/', $target ) );
+		$data['target'] = str_replace( home_url( '/', 'relative' ), '/', $target );
+		// Maybe add trailing slash if that is no search query.
+		if ( 0 !== strpos( $data['target'], '/?s=' ) ) {
+			$data['target'] = user_trailingslashit( $data['target'] );
+		}
 
 		// Trim target url.
 		if ( $wp_rewrite->permalink_structure ) {
-			$data['target'] = wp_parse_url( $data['target'], PHP_URL_PATH );
+			if ( 0 !== strpos( $data['target'], '/?s=' ) ) {
```

Comment:
Same as the above comment on `Statify_Dashboard::parse_target`. Are we sure, the `s` parameter is always first?

At this point we are working on the actual request target, so maybe we could just rely on `is_search()` here which should already be populated at this point.

Reply:
Changed that and introduced a new method that checks a query string for a search query.